packages/api/src/schema/api-schema.yaml

```diff
@@ -2052,7 +2015,97 @@ components:
           enum:
             - eip712
             - flow
-
+        storage:
+          additionalProperties: false
+          properties:
+            ipfs:
+              type: object
+              additionalProperties: false
+              properties:
+                $ref: "#/components/schemas/ipfs-file-info/properties"
+                updatedAt:
+                  readOnly: true
+                  type: number
+                  description: |
+                    Timestamp (in milliseconds) at which IPFS export task was updated
+                  example: 1587667174725
+            status:
+              $ref: "#/components/schemas/storageStatus"
+    ipfs:
```

Comment:
Can you rename this to `ipfs-export-params`?

Reply:
Updated